src/domains/transaction/pages/SendTransfer/SendTransfer.tsx

```diff
@@ -236,6 +236,11 @@ export const SendTransfer = () => {
 				setLastEstimatedExpiration(transactionInput.data.expiration);
 			}
 
+			if (activeWallet.isLedger()) {
```

Comment:
Why do we need to manually connect here? The SDK will connect when it actually needs a connection and not otherwise, see https://github.com/ArkEcosystem/platform-sdk/blob/master/packages/platform-sdk-ark/source/transaction.service.ts#L219-L220.

Reply:
Same as above, from the initial implementation, dw manages the connection status in its own hook. If you comment out these lines, the tests will fail to render the ledger review step which indicates this issue. We could revisit this implementation, which I think will require some extra effort and testing to make sure the existing functionality will not break.